qemu/tests/cfg/sev_basic_config.cfg

```diff
@@ -0,0 +1,16 @@
+- sev_basic_config:
+    type = sev_basic_config
+    kill_vm = yes
+    login_timeout = 240
+    image_snapshot = yes
+    mem = 4096
+    vm_secure_guest_type = sev
+    vm_sev_reduced_phys_bits = 1
+    vm_sev_cbitpos = 51
+    sev_keyword = "sev-es"
+    sev_enable_check = "journalctl|grep -i ${sev_keyword}"
```

Comment:
sev_enable_check = "journalctl|grep -wi {sev_keyword}"

use it by params['sev_enable_check'].format(sev_keyword='xxxx'), pls see comments below for the reason